src/Client.php

```diff
@@ -347,7 +347,7 @@ private function applyOptions(RequestInterface $request, array &$options)
         if (isset($options['query'])) {
             $value = $options['query'];
             if (is_array($value)) {
-                $value = http_build_query($value, null, '&', PHP_QUERY_RFC3986);
+                $value = http_build_query($value, false, '&', PHP_QUERY_RFC3986);
```

Comment:
If we're going to change this, then I think this should still be an empty string. See http://php.net/manual/en/function.http-build-query.php